packages/tools/devtools/devtools-core/package.json

```diff
@@ -48,8 +48,8 @@
 	"types": "lib/public.d.ts",
 	"scripts": {
 		"api": "fluid-build . --task api",
-		"api-extractor:commonjs": "flub generate entrypoints --outFileAlpha legacy --outDir ./dist",
-		"api-extractor:esnext": "flub generate entrypoints --outFileAlpha legacy --outDir ./lib --node10TypeCompat",
+		"api-extractor:commonjs": "flub generate entrypoints --outDir ./dist",
```

Comment:
This will automatically generate the `alpha` type-rollup based on the `alpha` entry under `exports`?

Reply:
Correct. It always would -- that is the default value for `--outFileAlpha`. We just shouldn't have added the argument to rename these.